tests/end_to_end/netsh_test.cpp

```diff
@@ -401,118 +401,6 @@ TEST_CASE("show verification droppacket.o", "[netsh][verification]")
                   "Program terminates within 0 loop iterations\n");
 }
 
-TEST_CASE("show verification xdp_adjust_head_unsafe.o", "[netsh][verification]")
```

Comment:
I don't think these netsh tests should be removed.  They shouldn't be using bpf2c.